modules/files/files.go

```diff
@@ -74,6 +84,11 @@ func CopyTerragruntFolderToTemp(folderPath string, tempFolderPrefix string) (str
 // CopyFolderToTemp creates a copy of the given folder and all its filtered contents in a temp folder
 // with a unique name and the given prefix.
 func CopyFolderToTemp(folderPath string, tempFolderPrefix string, filter func(path string) bool) (string, error) {
+	_, err := DirExistsE(folderPath)
```

Comment:
Shouldn't you be storing the boolean returned here and checking it?

Reply:
fixed to use the bool from FileExistsE